### PR TITLE
Add Token Classification Example for KerasNLP

### PR DESCRIPTION
**Pull request by shivance**
Closes [keras-nlp/#754](https://github.com/keras-team/keras-nlp/issues/754)

Using BERT to reduce the script running time, will change to RoBERTa later

### CODE REVIEW

File: examples/nlp/semantic_similarity_with_keras_nlp.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
+"""
```
Comment (mattdangerw) [inline examples/nlp/semantic_similarity_with_keras_nlp.py:1]: I think you pulled in your other guide by accident, you may want to rebase out the commits for this guide.

File: examples/nlp/token_classification.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+from datasets import load_dataset
```
Comment (mattdangerw) [inline examples/nlp/token_classification.py:1]: I think we can go ahead and start filling out the rough text for this guide anytime. So introduce the text classification task, talk about the dataset, why we need to preprocess it for the BERT subword tokenizer in particular.

Even if we end up tweaking the preprocessing routines a bit during review, the overall flow and intro description can probably stay fairly static!

**Comment by mattdangerw** [comment]
Thank you! @sampathweb and I will take a look, but left some high level comments for now.

**Comment by shivance** [comment]
I've opened an issue [here](https://github.com/keras-team/keras-nlp/issues/927) which elaborates the potential and clear cut roadmap for this example, and needs review from @mattdangerw and @sampathweb

File: examples/nlp/token_classification.py (hunk 2 above)
Comment (shivance) [reply examples/nlp/token_classification.py:1]: Sounds good @mattdangerw .

**Pull request closed without merging**